components/tools/ToolsList.js

```diff
@@ -6,7 +6,7 @@ export default function toolsList({ toolsData }) {
     <div className="" data-testid="ToolsList-main" >
       {Object.keys(toolsData).map((categoryName, index) => {
         if(toolsData[categoryName].toolsList.length > 0) return (
-        <div className='my-8' key={index} id={categoryName}>
+        <div className='my-8' key={index} id={categoryName} style={{scrollMargin: "100px"}}>
```

Comment:
You can't use inline styles to add styling inside `div`. Kindly use tailwind utility classes to add the CSS.